data/MLO/projects.yaml

```diff
@@ -490,3 +490,19 @@ projects:
       - PyTorch
     date_added: 2022-07-07
     date_updated: 2022-07-07
+
+  anomaly-detection:
+    name: Distributed Homomorphic Anomaly Detection
+    categories:
+      - Learning
+    applications:
+      - Info
+    description: Runs anomaly detection algorithms in a distributed and encrypted fashion
```

Comment:
The description should be at least 1 paragraph, better 2 paragraphs. It should describe:
- what problem it solves
- how it solves it
- why it's better than other solutions